lib/authentication.rb

```diff
@@ -0,0 +1,32 @@
+require "jwt"
+
+class Authentication
+  ALGORITHM = "HS256".freeze
+
+  class << self
+    def encode payload
+      payload[:exp] = 4.hours.from_now.to_i
```

Comment:
4 đưa vào setting đi em

Reply:
Luồng hoạt động
- Khi đăng nhập thành công (kiểm tra password đúng), phía server sẽ tạo chuỗi jwt trả về cho người dùng(sử dụng payload là thông tin muốn mã hóa như id, 1 secret key là 1 chuỗi khóa bảo mật và thuật toán dùng để mã hóa)
- Ở các request cần xác thực, người dùng cần gửi kèm chuỗi token thông qua header của request, server sẽ giải mã chuỗi token bằng secret key và thuật toán khi tạo mã, nếu token hợp lệ thì thực hiện request của người dùng